libs/shared/ui/multi-select.tsx

```
@@ -609,7 +615,14 @@ const MultipleSelector = React.forwardRef<MultipleSelectorRef, MultipleSelectorP
                                   )}
                                 >
 
-                                  {option.label}
+                                  <button
+                                    role="menu"
+                                    type="button"
+                                    aria-label={`command item ${option.value}`}
+                                  >
```

Biome (1.9.4)

[error] Interactive elements should not be assigned non-interactive roles.

WAI-ARIA roles should not be used to convert an interactive element to a non-interactive element.
Wrap your interactive element in a <div> with the desired role or put the content inside your interactive element.
Unsafe fix: Remove the role attribute.


(lint/a11y/noInteractiveElementToNoninteractiveRole)